extension/chrome/settings/index.ts

```diff
@@ -151,7 +151,12 @@ Catch.try(async () => {
       }
     }
 
-    await renderBlogPostList();
+    Api.retreiveBlogPosts().then((res: { url: string, date: string, title: string }[]) => {
+      for (const post of res) {
+        $('.blog_post_list').append(
+          `<div class="line"><a href="https://flowcrypt.com${Xss.escape(post.url)}" target="_blank">${Xss.escape(post.title.trim())}</a> (${Xss.escape(post.date.trim())})</div> \n`);
+      }
+    }).catch((e: Error) => Api.err.isSignificant(e) ? Catch.handleErr(e) : undefined);
```

Comment:
Don't clarify `e` as `Error` because in JS, anything could be thrown. Narrowing it down to `Error` gives you a false sense of security. I could be a string or a null, for all you know. Do: `.catch(e => Api.err.isSignificant(e) ? Catch.handleErr(e) : undefined);`